pontoon/api/schema.py

```diff
@@ -109,14 +111,17 @@ class Meta:
     )
 
     def resolve_localizations(obj, _info, include_disabled, include_system):
-        qs = obj.project_locale
+        projects = obj.project_locale.visible_for(_info.context.user)
```

Comment:
Nit: I guess `_info` should be `info` now that we actually use it?

Reply:
I couldn't find other places that follow this convention so I'm going to replace `_info` with `info`.